sdk/simulator-service/src/implementation/simulator-engine/reducer/importReducer.ts

```diff
@@ -0,0 +1,6 @@
+import { steps } from '@summerfi/sdk-common/simulation'
+import { ISimulationState } from '../../../interfaces/simulation'
+
+export function importReducer(_step: steps.ImportStep, state: ISimulationState): ISimulationState {
+  return state
```

Comment:
This is what I had to fix in my PR: the reducer is not returning the steps, so the steps list is empty
```suggestion
  return {
    ...state,
    steps: {
      ...state.steps,
      [step.name]: step,
    },
  }
```